app/controllers/participate/login.js

```diff
@@ -24,7 +24,7 @@ export default Ember.Controller.extend({
           this.transitionToRoute('participate.survey.consent');
         })
         .catch((e) => {
-          if (e.status === 404) {
+          if (e.status === 404 || e.status === 401) {
```

Comment:
It seems like there are two categories of error that we're trying to deal with here...

1. Authentication errors involving network requests
2. Transition errors involving ember

Rather then checking for individual status codes and event names, perhaps we could rearrange this conditional (or promise chain)? Otherwise, we'll just keep on modifying the conditional with more special cases every time someone triggers another kind of server error.

For a concrete example: when staging-metadata was down last night, the error code we saw was 500. It seems that would still not be handled by this code.